### PR TITLE
Add beanstalk config to our repos

### PR DESCRIPTION
**Pull request by andrewjhumphrey**
NOTE:  - .elasticbeanstalk/config.yml 
    will need to be santisied before it is released opensource.

Allows us to do deploys to the elastic beanstalk environment
http://mandrill-merge-dev.elasticbeanstalk.com/

### CODE REVIEW

**Comment by andrewjhumphrey** [comment]
It's not sensitive persay, it just leaks information (like key names and environment names) it doesn't give anyone access just information

File: README.elasticbeanstalk [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+3.  eb create --profile <fromabove> --region <fromabove>
+  - when prompted for Environment Name take the default 
+  - accept the default CNAME prefix
+
+4.  vist http://<appname>.elasticbeanstalk.com/ and profit.
+
+To deploy
+--------
+
+make sure you changes are comment
```
Comment (andrewjhumphrey) [inline README.elasticbeanstalk]: committed
*sigh*

**Comment by andrewjhumphrey** [comment]
Oh, we are closed but not merged?  Did it get done some other way?

**Comment by nocache** [comment]
I just deleted the branch (on github) since the repo is now public I
thought it would be better to be cautious and get rid of it for the moment.

On Thu, Apr 9, 2015 at 10:02 PM, Andrew Humphrey <notifications@github.com>
wrote:

> Oh, we are closed but not merged? Did it get done some other way?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/envato/mandrill-merge/pull/29#issuecomment-91241404>.
>



-- 

Glenn Tweedie  |  Developer  |  Envato Pty. Ltd.

E glenn.tweedie@envato.com  M +61 04 093 097 00

121 King Street, Melbourne, VIC 3000, Australia

http://envato.com

The email and any attachments are confidential and may contain information
that is exempt from disclosure by law and subject to copyright. If you have
received this message in error please let me know and delete it.

**Pull request closed without merging**